eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -455,6 +454,7 @@ jobs:
           customPreBuildArgs=''
           customBuildArgs=''
           extraBuildProperties=''
+          customBuildArgs="--ci --prepareMachine -c ${{ parameters.configuration }}"
```

Comment:
How is this change related?  Also why wasn't this just done two lines above?

Reply:
Copy / Paste error, I should have replaced L455 instead. It doesn't cause any issues but I will clean this up when I touch the file again. Thanks for pointing it out.

Unrelated change. I noticed that the Unix test invocation didn't receive these expected arguments. I didn't want to spin a new with PR just for this one liner to save resources.